Added ICO Template with basic instructions

cwitchger opened the pull request:
ICO Template for Java. I have tested it locally using the privatenet Docker image and the neo-python CLI tool. If there are any suggestions for improvements, let me know.

erikzhang commented on ico-template/README.md:
```diff
@@ -0,0 +1,16 @@
+To run this ICO Template, you must first setup [neo-devpack-java](https://github.com/neo-project/neo-devpack-java) and install the artifact to your local maven .m2 folder as well as your system's dotnet folder.
+
+**NOTE**
+There is a pull request in for a modification to neo-devpack-java that is required for running this ICO Template. Please make sure the Runtime class is updated with the following:
```
This PR(https://github.com/neo-project/neo-devpack-java/pull/14) has been merged already.

cwitchger replied:
I will remove the note. Good catch. Let me know if you find anything else.

erikzhang commented on ico-template/.gitignore:
```diff
@@ -0,0 +1,21 @@
+# Compiled class file
```
We don't need to have a ".gitignore" in every project. Please merge it into the root ".gitignore".

cwitchger commented:
Updated the root .gitignore file and deleted the one in the ico-template folder based on your suggestion. Let me know if there is anything else I can fix.
